stdlib/@python2/_typeshed/__init__.pyi

```diff
@@ -155,17 +145,10 @@ class SupportsNoArgReadline(Protocol[_T_co]):
 class SupportsWrite(Protocol[_T_contra]):
     def write(self, __s: _T_contra) -> Any: ...
 
-if sys.version_info >= (3,):
-    ReadableBuffer = Union[bytes, bytearray, memoryview, array.array[Any], mmap.mmap]
-    WriteableBuffer = Union[bytearray, memoryview, array.array[Any], mmap.mmap]
-else:
-    ReadableBuffer = Union[bytes, bytearray, memoryview, array.array[Any], mmap.mmap, buffer]
-    WriteableBuffer = Union[bytearray, memoryview, array.array[Any], mmap.mmap, buffer]
-
-if sys.version_info >= (3, 10):
-    from types import NoneType as NoneType
-else:
-    # Used by type checkers for checks involving None (does not exist at runtime)
-    @final
-    class NoneType:
-        def __bool__(self) -> Literal[False]: ...
+ReadableBuffer = Union[bytes, bytearray, memoryview, array.array[Any], mmap.mmap, buffer]
+WriteableBuffer = Union[bytearray, memoryview, array.array[Any], mmap.mmap, buffer]
+
+# Used by type checkers for checks involving None (does not exist at runtime)
```

Comment:
```suggestion
```
It does exist in Python 2

```
$ python2
Python 2.7.12 (default, Nov 12 2018, 14:36:49) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import types
>>> types.NoneType
<type 'NoneType'>
>>> 
```

Reply:
Yes, `types.NoneType` exists, but the comment is about `_typeshed.NoneType`.

Reply:
My reading comprehension needs some work sometimes :)

In that case the comment still doesn't make much sense, since none of this file exists at runtime.